frameworks/Rust/faf/faf.dockerfile

```diff
@@ -13,4 +13,4 @@ RUN RUSTFLAGS="-Ctarget-cpu=native -Ztune-cpu=native -Zmutable-noalias=yes -Clin
    && strip ./target/x86_64-unknown-linux-gnu/release/faf-ex
 
 EXPOSE 8089
-CMD ./target/x86_64-unknown-linux-gnu/release/faf-ex
+CMD nice -19 ./target/x86_64-unknown-linux-gnu/release/faf-ex
```

Comment:
Sorry,  Will you do that in your servers ??

Reply:
Hey, I'm not sure what you mean. Do it where? In the code itself?

Reply:
Nevermind, I'll move this into my code to simplify it